criterion-measurement/src/Criterion/Measurement.hs

```diff
@@ -119,7 +130,10 @@ data GCStatistics = GCStatistics
 --
 -- If you need guaranteed up-to-date stats, call 'performGC' first.
 getGCStatistics :: IO (Maybe GCStatistics)
-#if MIN_VERSION_base(4,10,0)
+#if defined(__GHCJS__)
+getGCStatistics = return Nothing
```

Comment:
Is there a GHCJS-specific reason why this can't be done? If so, can you leave a comment explaining why not?

Reply:
Yes, IIRC, the C-primitives to lookup RTS stats are simply not implemented in GHCJS C-shims.